types/index.d.ts

```diff
@@ -117,7 +117,6 @@ export * from '../Libraries/LayoutAnimation/LayoutAnimation';
 export * from '../Libraries/Linking/Linking';
 export * from '../Libraries/Lists/FlatList';
 export * from '../Libraries/Lists/SectionList';
-export * from '../Libraries/Lists/VirtualizedList';
```

Comment:
We are still exporting VirtualizedList from RN, so its types will need to be re-exported here still. I think we should keep the typetest for similar reason.

Reply:
I've just pushed a commit adding this back